src/driver/CMakeLists.txt

```diff
@@ -40,11 +40,11 @@ add_custom_command(
                 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/driver
         BYPRODUCTS ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/driver
 )
-set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/driver)
-rocm_clang_tidy_check(driver)
+set_directory_properties(PROPERTIES ADDITIONAL_CLEAN_FILES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/driver)
 
-target_link_libraries(driver migraphx_all_targets migraphx_onnx migraphx_tf)
+if(CLANG_TIDY_EXE)
+    rocm_clang_tidy_check(driver)
+endif()
 
-rocm_install_targets(
-  TARGETS driver
-)
+target_link_libraries(driver PRIVATE migraphx migraphx_onnx migraphx_tf)
+rocm_install_targets(TARGETS driver)
```

Comment:
This is missing a newline at the end. Also formatting changes to cmake should go into a seperate PR.